docs/7.x/changelog.md

```diff
@@ -33,22 +33,70 @@ These versions are past their End of Support date, and no longer receive securit
 and bug fixes. [Gravity customers](https://gravitational.com/gravity/demo/) can
 extend updates past End of Support through customer agreements if required.
 
-| Version             | Latest Patch | LTS | Release Date         | End of Support          | Kubernetes Version   | Teleport Version |
-| ------------------- | ------------ | --- | -------------------- | ----------------------- | -------------------- | ---------------- |
-| [6.3](#63-releases) | 6.3.18       | No  | December 18, 2019    | April 3, 2020 (7.0)     | 1.17.6               | 3.2.13           |
-| [6.2](#62-releases) | 6.2.5        | No  | September 24, 2019   | December 18, 2019 (6.3) | 1.16.3               | 3.2.13           |
-| [6.0](#60-releases) | 6.0.10       | No  | July 17, 2019        | August 2, 2019 (6.1)    | 1.14.7               | 3.2.12           |
-| [5.6](#56-releases) | 5.6.8        | No  | April 19, 2019       | July 17, 2019 (6.0)     | 1.14.7               | 3.0.6-gravity    |
-| [5.4](#54-releases) | 5.4.10       | No  | December 14, 2018    | March 8, 2019 (5.5)     | 1.13.5               | 2.4.10           |
-| [5.3](#53-releases) | 5.3.9        | No  | October 19, 2018     | December 14, 2018 (5.4) | 1.12.3               | 2.4.7            |
-| [5.2](#52-releases) | 5.2.18       | Yes | October 15, 2018     | October 15, 2019        | 1.11.9               | 2.4.10           |
-| [5.0](#50-releases) | 5.0.36       | Yes | April 18, 2018       | April 13, 2019          | 1.9.13-gravitational | 2.4.10           |
-| [4.x](#4x-releases) | 4.68.0       | Yes | June 1, 2017         | November 16, 2018       | 1.7.18-gravitational | 2.3.5            |
-| [3.x](#3x-releases) | 3.64.0       | Yes | February 16, 2017    | June 2, 2018            | 1.5.7                | 2.0.6            |
-| [1.x](#1x-releases) | 1.29.0       | Yes | November 2nd, 2016   | March 21, 2018          | 1.3.8                | 1.2.0            |
+| Version             | Latest Patch  | LTS | Release Date         | End of Support          | Kubernetes Version   | Teleport Version |
+| ------------------- | ------------- | --- | -------------------- | ----------------------- | -------------------- | ---------------- |
+| [7.1](#71-releases) | 7.1.0-alpha.5 | No  | N/A                  | N/A                     | 1.19.6               | 3.2.17-gravity   |
```

Comment:
I realize we're using a nomenclature like supported / unsupported versions, but I sort of think 7.1 belongs in the "active" releases table, and haven't really expired as an unsupported version.

Reply:
Moved to the top table in 8b6b022.